bool off_road
bool on_shoulder
bool off_route
bool wrong_way
bool reached_goal
bool not_moving
string[] collisions  # list of vehicle_ids
